### DISCUSSION TITLE
deno lambdas: debugging .ts arc sandbox via vscode

### DISCUSSION THREAD

**Discussion by martinsebastian** [beginner-corp/begin-community#61] [Q&A]
Hello sweethearts,

I am a bit lost to set up my vscode to debug my deno and jump into any breakpoint in my .ts files. Attaching to the node process (npm sandbox) doesn't seem to work, most likely because deno already compiled the ts files, and launching a deno debugger works neither because it requires to directly launch a ts entry point file with the deno runtime, which is a thing that @architect does under the hood, right?

How do you wonderful people do it? :)

Cheerio
Martin

**Comment by ryanblock** [comment]
Hey @martinsebastian! It's a question we've gotten a fair amount. To the "how do we do it?" part: tbqh, `console.log`. Works pretty well!

Re. debuggers, this issue may be of some help: https://github.com/architect/architect/issues/640 At some point sooner or later I'll get around to giving this a closer look, but in the mean time I remain open to PRs!
